drivers/bluetooth/hci/hci_silabs_efr32.c

```diff
@@ -52,6 +52,7 @@ static struct k_fifo slz_rx_fifo;
 /* FIXME: these functions should come from the SiSDK headers! */
 void BTLE_LL_EventRaise(uint32_t events);
 void BTLE_LL_Process(uint32_t events);
+int16_t BTLE_LL_SetMaxPower(int16_t power);
```

Comment:
I think you also need to add a stub to `modules/hal_silabs/simplicity_sdk/src/blob_stubs.c` to allow building without the binary blobs. There are some test cases that will likely fail to build otherwise, like this one: https://github.com/zephyrproject-rtos/zephyr/blob/e4c16b5433cdbf968f042976b0192f25914d4b96/samples/bluetooth/central/sample.yaml#L12

Reply:
Thanks, I forgot these. Fixed now.